cmd/approvelist/main.go

```diff
@@ -20,10 +20,11 @@ import (
 )
 
 const (
-	srcSuffix = "-original.xlsx"
-	dstSuffix = "-approve-list.xlsx"
-	pass      = "Godkjent"
-	fail      = "Ikke godkjent"
+	srcSuffix      = "-original.xlsx"
+	dstSuffix      = "-approve-list.xlsx"
+	pass           = "Godkjent"
+	fail           = "Ikke godkjent"
+	approvalColumn = "E"
```

Comment:
Would it make sense to instead calculate this by reading the header line?

Reply:
Alternate solution provided in 51f3e1d.